tests/test_openjdk.py

```diff
@@ -67,28 +67,35 @@ def test_jdk_version(container, java_version):
     indirect=["container_per_test"],
 )
 @pytest.mark.parametrize(
-    "test_to_run, expected_strings, expected_exit_status, java_params",
+    "test_to_run, expected_strings, expected_exit_status, java_params, arguments, environment",
```

Comment:
Why are you adding additional parameters that you are not even using?

Reply:
I will wait for the resolution in https://github.com/SUSE/BCI-tests/pull/157

Reply:
Once #157 the same solution has being used here